zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/internal/client/HttpCall.java

```diff
@@ -188,10 +248,10 @@ V parseResponse(AggregatedHttpResponse response, BodyConverter<V> bodyConverter)
       if (status.codeClass().equals(HttpStatusClass.SUCCESS)) {
         return null;
       } else if (status.code() == 404) {
-        throw new FileNotFoundException(request.path());
+        throw new FileNotFoundException(request.headers().path());
```

Comment:
the fact that armeria headers includes path always confuses me. I wonder if `requestLine()` would have been in hindsight a better term

Reply:
Request line is something like `POST /`. I'm not sure if users usually deal with the combined request line like that, method and path are often separate, like in the servlet API - https://docs.oracle.com/javaee/6/api/javax/servlet/http/HttpServletRequest.html

Is the confusion because you would expect headers to only be headers, without the request line? That makes some sense, but not sure I want to spend brainpower thinking too much about it ;)